automation_tools/__init__.py

```diff
@@ -1809,8 +1828,18 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
         The url for capsule repo from latest satellite compose.
     TOOLS_URL
         The url for sat-tools repo from latest satellite compose.
+    FROM_VERSION:
+        The satellite current version. To differ representation of
+        Default Organization. e.g '6.1','6.0'
 
     """
+    if os.environ.get('FROM_VERSION') == '6.1':
+        org = '\'Default Organization\''
```

Comment:
Because 6.1 has 'Default Organization' and 6.0 has 'Default_Organization'. There is space between two words in current 6.1 version. And Hence while passing to hammer command I should pass it with quotes. So that hammer wont treat it as arguments.
e.g : 
With Quotes:
```
[root@sat6-qe-rhel72 ~]# hammer -u admin -p changeme product list --organization 'Default Organization'
---|------------------------------------------------------------|-------------|----------------------|--------------|------------------
ID | NAME                                                       | DESCRIPTION | ORGANIZATION         | REPOSITORIES | SYNC STATE       
---|------------------------------------------------------------|-------------|----------------------|--------------|------------------
15 | capsule6_latest                                            |             | Default Organization | 1            | Syncing Complete.
12 | Red Hat Beta                                               |             | Default Organization | 0            |                  
```
And without Quotes:
```
[root@sat6-qe-rhel72 ~]# hammer -u admin -p changeme product list --organization Default Organization
Error: too many arguments

See: 'hammer product list --help'
```